client/src/components/EditProfile/EditPhoto/useStyles.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  authWrapper: {
+    display: 'flex',
+    alignItems: 'flex-start',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    minHeight: '100%',
+    paddingBottom: 40,
+  },
+  title: {
+    padding: 40,
+  },
+  reminder: {
+    padding: 20,
+    width: '16vw',
+  },
+  centerRow: {
```

Comment:
Since we've switched to a Box component above, you can add these styles it it directly :)

Reply:
It is used for several components. May I keep it?

Reply:
You can keep it but I think it's a good practice to apply significant styles directly to the relevant components :)

Reply:
I agree. I fixed it.